app/router.js

```diff
@@ -9,5 +9,6 @@ export default class Router extends EmberRouter {
 Router.map(function() {
   this.route('profile', { path: '/profile/:profile_id' }, function () {
     this.route('connections');
+    this.route('education');
```

Comment:
We should also setup scenarios of nested routes and routes with `resetNamespace: true` in another PR.

Reply:
Good point